README.md

```diff
@@ -342,7 +343,14 @@ greentea_parse_kv(key,value)   |                   |
 greentea_parse_kv              |  {{__sync;UUID}}  |
 echoes __sync event with       |------------------>|
 the same UUID to master        |                   |
+                               |                   |self.send_kv("__host_ack", UUID)
+                               |{{__host_ack;UUID}}|<-----------------------------
+                               |<------------------|
+                               |                   |
                                |                   |
+greentea_parse_kv              |{{__host_ack;UUID}}|
+echoes __host_ack event        |------------------>|
+to master                      |                   |
```

Comment:
While this may happen, it's not implemented in this PR or required. Perhaps we should simply drop it from this diagram?

Reply:
Actually come to think of it, we should drop all the changes in readme for this PR and add them with the greentea PR. If you think of it, nothing in this PR affects how greentea and htrun works. Thoughts?

Reply:
I think its fine for most of it to stay here, just because that's where its historically been. The greentea-client inside the OS doesn't mention all the different keys and such.